packages/here-wallet/src/lib/selector.ts

```diff
@@ -1,17 +1,16 @@
-import type { NetworkId } from "@near-wallet-selector/core";
-import { HereWallet, waitInjectedHereWallet } from "@here-wallet/core";
-
+import {
+  HereWallet,
+  waitInjectedHereWallet,
+  WidgetStrategy,
+} from "@here-wallet/core";
 import type { SelectorInit } from "./types";
 
 export const initHereWallet: SelectorInit = async (config) => {
-  const { store, logger, emitter, options, defaultProvider, defaultStrategy } =
-    config;
-
-  const here = new HereWallet({
-    networkId: options.network.networkId as NetworkId,
-    nodeUrl: options.network.nodeUrl,
-    defaultProvider,
-    defaultStrategy,
+  const { store, logger, emitter, options } = config;
+  const here = await HereWallet.connect({
+    defaultStrategy: new WidgetStrategy({
+      widget: "http://localhost:4173/connector/index.html",
```

Comment:
Maybe we should leave the [default URL](https://my.herewallet.app/connector/index.html)?

Reply:
yes, my bad